src/db/dbBPin.cpp

```diff
@@ -188,6 +188,8 @@ void dbBPin::setPlacementStatus(dbPlacementStatus status)
 {
   _dbBPin* bpin        = (_dbBPin*) this;
   bpin->_flags._status = status.getValue();
+  _dbBlock* block = (_dbBlock*) bpin->getOwner();
+  block->_flags._valid_bbox=0;
```

Comment:
= false  (its really a boolean field)

Reply:
actually it is of uint type and has been treated as such in the whole project
![Screenshot from 2020-08-10 19-02-04](https://user-images.githubusercontent.com/56893454/89809775-55123a00-db3c-11ea-8b53-11888313921c.png)